### PR TITLE
ACS-6736: Add integration test for bulk ingester events [live-ingester]

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/bulk_ingester/BulkIngesterEventIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                  "modifierId": "hr_user",
+                  "aspectNames": [
+                    "cm:indexControl",
+                    "cm:auditable"
+                  ],
+                  "contentInfo": {
+                    "contentSize": 330,
+                    "encoding": "ISO-8859-1",
+                    "mimetype": "text/xml"
+                  },
```
Comment (Pawel-608) [inline live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/bulk_ingester/BulkIngesterEventIntegrationTest.java:99]: I am wondering if we actually need this content-info data? Maybe I could change it to "hasContent" boolean? wdyt @tpage-alfresco, @krdabrowski?

Comment (krdabrowski) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/bulk_ingester/BulkIngesterEventIntegrationTest.java:92]: We don't pass this data further to HxI?

**Review by krdabrowski — APPROVED** [review]
LGTM

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/bulk_ingester/BulkIngesterEventIntegrationTest.java (hunk 1 above)
Comment (Pawel-608) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/bulk_ingester/BulkIngesterEventIntegrationTest.java:92]: no, we just need info if node contains content or not - I wonder if we will use it in the future 🤔

Comment (tpage-alfresco) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/bulk_ingester/BulkIngesterEventIntegrationTest.java:92]: We could potentially use the mimetype in the future to determine whether a transformation is possible. Currently we're just sending everything for transformation.

**Pull request merged**